README.md

```diff
@@ -88,4 +88,18 @@ If you install Vagrant, you can run the application in its own vm. Just install
 ```bash
 $ vagrant up
 
-```
\ No newline at end of file
+## Running in Docker
+If you want to build and run a docker image of this application you can run:
+```bash
+$ docker build --tag todo-app .
```

Comment:
When you build the image, you should specify whether you want the "production" or "development" stage:
E.g.:
`docker build --tag todo-app --target production .`